lib/floe/workflow/runner/kubernetes.rb

```diff
@@ -102,6 +102,33 @@ def cleanup(runner_context)
           delete_secret(secret) if secret
         end
 
+        def wait(timeout: nil, events: %i[create update delete])
```

Comment:
Out of curiosity was this pattern pulled from one of the providers?  I'm just curious how you knew to handle things like the timeouts happening at interesting times.